### PR TITLE
update users feed, including likes in the DTO, and some bug fixes 

### PR DESCRIPTION
**Pull request by MohammedLilo**

### CODE REVIEW

File: Models/User.cs
Comment (KAJOiq) [inline Models/User.cs:1]: Great job!

**Comment by KAJOiq** [comment]
Please resolve conflict and then push your code again! NIGGAAAAAAAAAAAA

File: Controllers/Web/v1/UsersController.cs
Comment (KAJOiq) [inline Controllers/Web/v1/UsersController.cs:1]: Great Job!

**Review by diyar214 — COMMENTED** [review]
Require testing

**Review by KAJOiq — APPROVED** [review]

**Pull request merged**